src/components/chips/js/chipsDirective.js

```diff
@@ -224,7 +226,6 @@
         readonly: '=readonly',
         removable: '=mdRemovable',
         placeholder: '@',
-        mdEnableChipEdit: '@',
```

Comment:
Good point - it's because the scope binding is never used and we are only checking the attribute once by using `$attrs`.
> https://github.com/DevVersion/material/blob/3f20904c387e4c7154d67a73d5a8f94213a3607e/src/components/chips/js/chipsController.js#L57